connectivity/check/logging.go

```diff
@@ -285,7 +285,7 @@ func (t *Test) Infof(format string, a ...interface{}) {
 	t.logf(info+" "+format, a...)
 }
 
-func (t *Test) failCommon() {
+func (t *Test) failCommon(ctx context.Context) {
```

Comment:
This `ctx` is not used after being passed in. It seems like the `context.Background()` needs to be replaced with it on line 298.